test/test.js

```diff
@@ -455,7 +455,7 @@ describe("S3rver Tests", function() {
             (err, object) => {
               if (err) done(err);
               object.Metadata.should.have.property("somekey", "value");
-              object.ContentType.should.equal("image/jpeg");
+              object.ContentType.should.equal("application/octet-stream");
```

Comment:
Did you check this behaviour on s3?

I am not 100% sure if s3 tries to guess the header or not before adding `application/octet-stream`.